sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -12,7 +12,7 @@ create, manage, and deploy public and private SSL/TLS certificates
 
 [Source code][key_client_src] | [Package (PyPI)][pypi_package_keys] | [API reference documentation][reference_docs] | [Product documentation][keyvault_docs] | [Samples][key_samples]
 
-## Getting started
+## Getting started2
```

Comment:
Looks like a typo? started2

Reply:
Yeah I did that to make sure the README verification would work with the new omit rules @scbedd is adding to docsettings.yml